initfiles/sbin/CMakeLists.txt

```diff
@@ -37,6 +37,15 @@ ADD_CUSTOM_TARGET(ProcessFiles-initfiles-sbin ALL DEPENDS ${outFiles} ${noInstal
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/regex.awk.in.cmake" "${CMAKE_CURRENT_BINARY_DIR}/regex.awk")
 configure_file("${CMAKE_CURRENT_SOURCE_DIR}/backup.sh.in" "${CMAKE_CURRENT_BINARY_DIR}/backup.sh" @ONLY)
 
+if(USE_SPARK)
```

Comment:
we also need to configure HPCC-Platform/initfiles/etc/DIR_NAME/environment.xml.in
 if USE_SPARK, to add stock "mysparkthor" associated with "mythor" and sparkmaster="myesp/Instance"

Reply:
It is now included in the default environment.xml